files/private-chef-cookbooks/private-chef/libraries/chef_run_provider.rb

```diff
@@ -11,6 +11,7 @@ class ChefRun < Chef::Provider::LWRPBase
           mutate_chef_config
           converge
           Chef::Config.restore(old_config)
+          Chef::Cookbook::FileVendor.instance_variable_set(:@initialization_options, Chef::Config[:cookbook_path])
```

Comment:
:+1: though having to make this aware of the internals of FileVendor makes me itch a bit.